stable_baselines3/common/logger.py

```diff
@@ -336,7 +362,10 @@ def write(self, key_values: Dict[str, Any], key_excluded: Dict[str, Union[str, T
                 continue
 
             if isinstance(value, np.ScalarType):
-                self.writer.add_scalar(key, value, step)
+                if isinstance(value, str):
```

Comment:
a `str` is a `np.ScalarType` ?

Reply:
I was surprised too. But [yes](https://stackoverflow.com/questions/21968643/what-is-a-scalar-in-numpy). I double checked this myself.

Reply:
interesting, but for clarity the if could be moved outside, no? (and replace the if by a elif for the scalar type)
or at least add a comment ;)

Reply:
I did not want to move it outside using elif in case someone at a later point changes the ordering of the elifs without realizing that `str` is a `np.ScalarType`.

Reply:
Added a comment in 8dbe360b84a88cadbae7ee6fb319aec823ffdf53